api/api/clubs/permissions.py

```diff
@@ -16,6 +16,13 @@ def has_object_permission(self, request, view, obj):
 class IsCoreMemberOrAdmin(permissions.BasePermission):
     """Custom permission to allow only core members of respective club and admins"""
 
+    def has_permission(self, request, view):
+        user_roles = request.user.roles.filter(Q(name='Coordinator') | Q(
+            name='Co-Coordinator') | Q(name='Core member'))
```

Comment:
Can we update these strings to variables from model here and other places in this file?